docs/2_4_common_schema.adoc

```diff
@@ -1150,15 +1148,6 @@ If <<initial>> is not present, its value is defined by <<table-definition-initia
 ^|[.underline]#calculated#, exact, approx
 ^|[.underline]#calculated#, exact, approx
 ^|--
-
-^|<<Clock,`Clocks`>>
-^|--
-^|--
-^|--
-^|[.underline]#exact#
-^|[.underline]#calculated#
-^|[.underline]#calculated#
-^|--
 |====
 
 _[Note: For local and output variables and <<initial>> = <<exact>>, then the variable is explicitly set in <<InitializationMode>>._
```

Comment:
Maybe ad a second note to be explicit

_[Note2: Clocks do not have initial attribute]_

Reply:
I added a normative sentence to that effect - please check.